OpenCanvas-LMS/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenCanvas Installation on Windows Azure
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fgopikap%2Fazure-quickstart-templates%2Fmaster%2FOpenCanvas-LMS%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fgopikap%2Fazure-quickstart-templates%2Fmaster%2FOpenCanvas-LMS%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+# OpenCanvas template 
+
+This template deploys a OpenCanvas to the ubuntu VM 16.04
+* Deploys on a Ubuntu VM 16.04
+* The template installs postgres9.5 version and a new database is created
+
+
+### How to Deploy
+You can deploy the template with Azure Portal, or PowerShell, or Azure cross platform command line tools.
+
+### Parameters to provide while deploying
+* You need to provide admin username and password for the OpenCanvas admin.
+* You need to provide the password for the postgres user.
+* The domainname have to be provided while deploying
+* The email to use for mail services(smtp services)
+
+### Configure smtp settings in outgoing_mail.yml file
```

Comment:
Move this to the deployment template so the step can be completed automatically, rather than a manual post deployment step.  Will review the rest of the code once this is done.